src/register/RegistrationFailure.jsx

```diff
@@ -28,8 +28,9 @@ const RegistrationFailureMessage = (props) => {
      userErrors.push(serverError);
      break;
 
-     default:
-        Object.keys(errorMessage).forEach((key) => {
+    default:
+      Object.keys(errorMessage).forEach((key) => {
+        if (key !== 'error_code') {
```

Comment:
Do we have backend cases where the key fields has error_code?  Does this inject error_code into fields?
https://github.com/edx/edx-platform/pull/26821/files?file-filters%5B%5D=.py#diff-edb3995ebb3452c122e32954149f176d38476d0c6b960164d09ff48f8d9a5f51R657

Reply:
In the response dictionary it saves a new key (error_code) with it's value set to the type of error.